src/components/search/data/tests/hooks.test.js

```diff
@@ -0,0 +1,129 @@
+import { renderHook } from '@testing-library/react-hooks';
+import '@testing-library/jest-dom/extend-expect';
+
+import {
+  useDefaultSearchFilters,
+  useRefinementsFromQueryParams,
+  useActiveRefinementsByAttribute,
+  useActiveRefinementsAsFlatArray,
+  useNbHitsFromSearchResults,
+} from '../hooks';
+
+const TEST_ENTERPRISE_UUID = 'test-enterprise-uuid';
+const TEST_SUBSCRIPTION_CATALOG_UUID = 'test-subscription-catalog-uuid';
+
+jest.mock('react-router-dom', () => ({
+  useLocation: () => ({
+    search: '?q=test%20query&subjects=Computer%20Science,Communication&availability=Upcoming&ignore=true',
+  }),
+}));
+
+describe('useRefinementsFromQueryParams hook', () => {
+  test('returns expected refinements data given specific query params', () => {
+    const { result } = renderHook(() => useRefinementsFromQueryParams());
+
+    const refinements = result.current;
+
+    // note: the query parameters are configured above when mocking out `useLocation`
+    expect(refinements).toEqual({
+      q: 'test query',
+      subjects: ['Computer Science', 'Communication'],
+      availability: ['Upcoming'],
+    });
+  });
+});
+
+describe('useActiveRefinementsByAttribute and useActiveRefinementsAsFlatArray hooks', () => {
+  const items = [{
+    attribute: 'subjects',
+    items: [{ label: 'Computer Science' }, { label: 'Communication' }],
```

Comment:
You end up using these same strings (`'Computer Science`) over and over in the tests; I'm inclined to make them variables...

Reply:
Updated to make the hardcoded strings as variables instead.